kitchen.vagrant.yml

```diff
@@ -1,71 +1,24 @@
-<%
-require 'time';
-%>
 ---
 driver:
   name: vagrant
   customize:
     natdnsproxy1: "on"
     natdnshostresolver1: "on"
 
```

Comment:
These may or may not be needed now that we are off the MITRE proxy - they don't hurt but ...